### ISSUE TITLE
MDX_Vault\MDX_Vault DX7II\Collections\STUDIOREINE

### ISSUE THREAD

**Issue by Piattica** [BobanSpasic/MDX_Vault#16]
many performances are silent

**Comment by BobanSpasic** [comment]
The complete conversion is gone wrong for this bank. The voice data is wrong.

**Comment by BobanSpasic** [comment]
Nope, I was wrong - the conversion is OK. I've manually checked the data.
Could you, please, write the names of the performances that are silent?

**Comment by Piattica** [comment]
Hi Boban

during the week end I retry loading and I let you know better one by one

Take care



Paolo Piattica

Il mar 5 dic 2023, 19:52 Boban Spasic ***@***.***> ha scritto:
